utils/log.go

```diff
@@ -90,12 +95,27 @@ type SimpleLogger interface {
 	Infow(msg string, keysAndValues ...any)
 	Warnw(msg string, keysAndValues ...any)
 	Errorw(msg string, keysAndValues ...any)
+	Tracew(msg string, keysAndValues ...any)
 }
 
 type ZapLogger struct {
 	*zap.SugaredLogger
 }
 
+const (
+	TraceLevel = zapcore.Level(-2)
```

Comment:
Do you think we can make it private? I don't think we might pass somewhere this value explicitly outside of this package